src/applications/vaos/tests/new-appointment/components/ConfirmationPage/ConfirmationRequestInfo.unit.spec.jsx

```diff
@@ -28,32 +37,30 @@ describe('VAOS <ConfirmationRequestInfo>', () => {
     };
     const pageTitle = 'Your appointment request has been submitted';
 
-    const tree = mount(
+    const screen = renderWithStoreAndRouter(
       <ConfirmationRequestInfo
         data={data}
         facilityDetails={facilityDetails}
         pageTitle={pageTitle}
       />,
+      {
+        initialState,
+        reducers,
+      },
     );
 
-    const text = tree.text();
-
-    const heading = tree.find('h1');
-
-    expect(text).not.to.contain('Community Care');
-    expect(text).to.contain('CHYSHR-Sidney VA Clinic');
-    expect(text).to.contain('Cheyenne, WY');
-
-    expect(tree.find('AlertBox').exists()).to.be.true;
-    expect(tree);
-    expect(heading.exists()).to.be.true;
-    expect(heading.text()).to.equal(pageTitle);
-
-    tree.unmount();
+    screen.getByText(/Your appointment request has been submitted./i);
```

Comment:
Looks like they recommend wrapping these in assertions: https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-get-variants-as-assertions